test/cag-various.js

```diff
@@ -14,3 +25,36 @@ test('CAG getOutlinePaths should work correctly', t => {
   t.deepEqual(cag.sides.length, 35)
   t.deepEqual(cag.isCanonicalized, true)
 })
+
+test.failing('CAG overCutInsideCorners should work correctly', t => {
+// test CAG without inside corners
+  const cag1 = CAG.rectangle({radius: [5,3]})
+  let res1 = cag1.overCutInsideCorners(1)
+
+  //t.deepEqual(cag1, res1) // not possible due to tags
+  t.deepEqual(compareCagSide(res1.sides[0]), {pos: [[-5, 3], [-5, -3]]})
+  t.deepEqual(compareCagSide(res1.sides[1]), {pos: [[-5, -3], [5, -3]]})
+  t.deepEqual(compareCagSide(res1.sides[2]), {pos: [[5, -3], [5, 3]]})
+  t.deepEqual(compareCagSide(res1.sides[3]), {pos: [[5, 3], [-5, 3]]})
+
+  const radius = 10
+  const cag2 = CAG.fromPoints([
+                               [radius,radius],
+                               [radius,-radius],
+                               [0,-radius],
+                               [0,0],
+                               [-radius,0],
+                               [-radius,radius],
+                             ])
+//console.log(cag2.toString())
+  let res2 = cag2.overCutInsideCorners(1)
+//console.log(res2.toString())
+// FIXME order of sides is just WRONG!!!
```

Comment:
why have it fail because of order ? I think we need to add a seperate test that demonstrates wrong ordering ?